geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommand.java

```diff
@@ -170,6 +171,8 @@ public ResultModel createRegion(
           help = CliStrings.CREATE_REGION__OFF_HEAP__HELP) Boolean offHeap,
       @CliOption(key = CliStrings.CREATE_REGION__PARTITION_RESOLVER,
           help = CliStrings.CREATE_REGION__PARTITION_RESOLVER__HELP) String partitionResolver,
+      @CliOption(key = CliStrings.CREATE_REGION__PARTITION_LISTENER,
+          help = CliStrings.CREATE_REGION__PARTITION_LISTENER__HELP) String partitionListener,
```

Comment:
I think a better example is the `cacheListner` parameter, it uses ClassName[] as the type. Since the listener is a `Declarable`, the `ClassName` type allows the user to specify the initialization values.

Reply:
OK - I changed that